src/Discourse/Event/DiscoursePost.php

```diff
@@ -46,6 +42,11 @@ public function isValidForSlack(): bool
 
         $post = $this->payload['post'];
 
+        // Only report NEW topics, not comments on existing topics
+        if (isset($post['id']) && $post['id'] > 1) {
+            return false;
+        }
+
```

Comment:
I think this should be extract refactored into private function for better readability, also would allow separate test for private func.

Reply:
I disagree; the method this occurs in _is_ the method for determining if a post should be ignored or not, and this is only _one_ criteria. Separating it for these three lines of code is overkill.

I likely _should_ have separate tests for the entities, but this logic is implicitly tested and covered with a number of other tests currently. We can do a QA sprint at some point to test each event individually.

Reply:
Disregard. I looked at the usage. This is fine already